guides/advanced_guides/asynchronous_updates.md

```diff
@@ -77,3 +77,23 @@ Failing to upload document:
   "processedAt": "2019-12-07T20:23:50.157436246Z"
 }
 ```
+
+### Killing MeiliSearch When a Task is Processing
+
+Since in MeiliSearch asynchronous task are [atomic](https://en.wikipedia.org/wiki/Atomicity_(database_systems)), killing MeiliSearch in the middle of the process does not corrupt or alter the database.
+
+> An atomic transaction is an indivisible and irreducible series of database operations such that either all occur, or nothing occurs.
```

Comment:
I think we should add this line to the glossary instead of linking to Wikipedia.